### ISSUE TITLE
Could we use tensorboard to visualize the model?

### ISSUE THREAD

**Issue by yaxi333** [tianweiy/CenterPoint-KITTI#7]
Hi,

I have noticed that you have used the tensorboard to save and plot the loss and learning_rate as follows:

![image](https://user-images.githubusercontent.com/57478405/123964704-0370d900-d9b4-11eb-82da-92e18e28dc58.png)

So I want to ask have you ever used the tensorboard to also save and visualize the details of layers, weights and biases of the model?

**Comment by tianweiy** [comment]
details of layers: You can just print the network and it will show the layers

weights and biases: I never visualize them yet
